src/Browsershot.php

```diff
@@ -716,6 +728,12 @@ protected function createCommand(string $url, string $action, array $options = [
 
         $command['options']['args'] = $this->getOptionArgs();
 
+        if (! empty($this->postParams) &&
```

Comment:
This conditional is unreadable to me. Could you extract this to a function that returns a boolean. The function should use early returns to break up the conditional

Reply:
I am sorry for not following your recommendations. I rethought the logic and figured out how to simplify it, at the same time adding the necessary, in my opinion, check for the presence of a URL scheme.